pkg/exporter/kafka_proto.go

```diff
@@ -28,6 +28,17 @@ func (kp *KafkaProto) ExportFlows(input <-chan []*flow.Record) {
 	}
 }
 
+func getFlowKey(record *flow.Record) []byte {
+	for k := range record.Id.SrcIp {
+		if record.Id.SrcIp[k] < record.Id.DstIp[k] {
+			return append(record.Id.SrcIp[:], record.Id.DstIp[:]...)
+		} else if record.Id.SrcIp[k] > record.Id.DstIp[k] {
```

Comment:
maybe just check for this condition since `==` and `<` both appends the same so we can avoid if else ?

Reply:
No, if both bytes are equal we first want to iterate another time. But we return as soon as there is a difference.

Reply:
ok Thanks for the explaining it